src/OpenLoco/Interop/Hook.cpp

```diff
@@ -231,6 +231,16 @@ namespace OpenLoco::Interop
         writeMemory(address, data, sizeof(data));
     }
 
+    void writeLocoCall(uint32_t address, uint32_t fnAddress)
+    {
+        uint8_t data[5] = { 0 };
+        data[0] = 0xE8; // CALL
```

Comment:
fwiw you can write `uint8_t data[5] = { 0xE8 };` and it will be exactly the same as these 2 lines (ie the end result will be `[0xE8, 0, 0, 0, 0]`